lib/index.js

```diff
@@ -113,6 +116,14 @@ exports.publish = function publish(basePath, config, callback) {
     return;
   }
 
+  if (options.cname) {
+    fs.writeFileSync(path.join(basePath, 'CNAME'), options.cname);
+  }
+  
+  if (options.nojekyll) {
+    fs.createFileSync(path.join(basePath, '.nojekyll'));
+  }
```

Comment:
Same above as above about writing to the `dest` directory instead of the root.  In addition, won't this file be ignored unless the `--dotfiles` option is also specified?